cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -589,7 +590,46 @@ def test_attribute_override():
     Bugzilla:
         1651099
     """
-    pass
+    attributes = [
+        ("class", "Request"),
+        ("instance", "TestNotification"),
+        ("message", "digitronik_msg"),
+        ("namespace", "/System"),
+    ]
+    req = "call_instance_with_message"
+    executions = {"simulation": "create", "execution": "digitronik_msg"}
+
+    collection = appliance.collections.button_groups
+    group = collection.create(
+        text="grp_{}".format(fauxfactory.gen_alphanumeric(3)),
+        hover="hover_{}".format(fauxfactory.gen_alphanumeric(3)),
+        type=getattr(collection, "PROVIDER"),
+    )
+    request.addfinalizer(group.delete_if_exists)
+
+    button = group.buttons.create(
+        text="btn_{}".format(fauxfactory.gen_alphanumeric(3)),
+        hover="hover_{}".format(fauxfactory.gen_alphanumeric(3)),
+        system="Request",
+        request=req,
+        attributes=attributes,
+    )
+    request.addfinalizer(button.delete_if_exists)
+
+    # Check msg override for simulation and actual execution
+    for exe, msg in executions.items():
+        pattern = "[miqaedb:/System/Request/TestNotification#{msg}]".format(msg=msg)
+        log = LogValidator("/var/www/miq/vmdb/log/automation.log", matched_patterns=[pattern])
+        log.fix_before_start()
+
+        if exe == "simulation":
+            button.simulate(provider.name, request="call_instance_with_message")
+        else:
+            view = navigate_to(provider, "Details")
+            custom_button_group = Dropdown(view, group.hover)
```

Comment:
Can this `Dropdown` be added in a parametrized view on the `provider` `Details` page? I have not seen widgets instantiated in tests before.

Reply:
Agreed with this... we thought same while the implementiong custom button testing arch. Then, I thought it should be TODO for me after complete every use cases. Its `inline` declaration of a `widget` and nothing bad in this. Custom button covers 28+ entities toolbars (All and Details) even I was getting a problem with the text while covering all entities.
I want similarity in custom button test.

Reply:
Good thought @john-dupuy, I think you're right that we can just have a parametrized widget on the page that resolves a `Dropdown`

The direct instantiation here works because @digitronik is passing the view to the call, so `__new__` resolves the actual widget, and not just the `WidgetDescriptor`.